iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/relational/analyzer/Analysis.java

```diff
@@ -123,6 +124,8 @@ public class Analysis implements IAnalysis {
 
   private final Map<NodeRef<Table>, Query> namedQueries = new LinkedHashMap<>();
 
+  private With with;
```

Comment:
add some comments about this important field, like what's this field used for? in which case

Reply:
Done